package.json

```diff
@@ -7,24 +7,26 @@
     "url": "https://github.com/jrrucker/elm-gallery.git"
   },
   "scripts": {
-    "start": "brunch watch --server",
-    "build": "brunch build --production"
+    "start": "parcel app/index.html",
+    "build": "parcel build app/index.html"
   },
   "devDependencies": {
-    "auto-reload-brunch": "^2.7.1",
-    "brunch": "^2.10.12",
     "download": "^6.2.5",
-    "elm": "^0.18.0",
-    "elm-fancy-brunch": "^1.0.0",
     "fs-extra": "^5.0.0",
-    "image-size": "^0.6.2"
+    "image-size": "^0.6.2",
+    "node-elm-compiler": "^5.0.1",
+    "parcel-bundler": "git+https://github.com/benthepoet/parcel.git#elm",
+    "parcel-plugin-asset-copier": "^1.0.0"
```

Comment:
These changes update the build to use a branch of parcel that supports Elm. 

The downside is that the static asset copying plugin doesn't watch the static asset directory for changes, so you have to restart the server if those files get edited. Luckily, we don't edit the album fixtures very often. I might open a PR for the plugin if it gets too annoying though.